iac-extensions/arm/src/main/java/org/sonar/iac/arm/tree/api/bicep/ObjectType.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * SonarQube IaC Plugin
+ * Copyright (C) 2021-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.iac.arm.tree.api.bicep;
+
+import java.util.List;
+import org.sonar.iac.arm.tree.api.ArmTree;
+
+public interface ObjectType extends ArmTree {
+  List<ArmTree> properties();
```

Comment:
I like the idea of simplifying the grammar like this. For me all arguments are valid. It's not following the official grammar, but it does not introduce additional depth. In case it dan dome to some conflicts in parsing or tree processing, we can add the dedicated tree later on.